graph-builder/src/graph.rs

```diff
@@ -98,12 +101,27 @@ pub fn register_metrics(registry: &prometheus::Registry) -> Fallible<()> {
 pub async fn index(
     req: HttpRequest,
     app_data: actix_web::web::Data<State>,
+) -> Result<HttpResponse, GraphError> {
+    GRAPH_INCOMING_REQS.with_label_values(&["/graph"]).inc();
+    _index(req, app_data).await
+}
+
+/// Serve Cincinnati graph requests legacy v1.
+pub async fn v1_index(
+    req: HttpRequest,
+    app_data: actix_web::web::Data<State>,
+) -> Result<HttpResponse, GraphError> {
+    GRAPH_INCOMING_REQS.with_label_values(&["/v1/graph"]).inc();
+    _index(req, app_data).await
+}
+
+async fn _index(
+    req: HttpRequest,
+    app_data: actix_web::web::Data<State>,
 ) -> Result<HttpResponse, GraphError> {
     let span = get_tracer().start("index");
     let _active_span = mark_span_as_active(span);
 
-    V1_GRAPH_INCOMING_REQS.inc();
```

Comment:
nit: I'd expect you could pull the endpoint off `req` without sharding out into `index` and `v1_index`, but I don't have a problem with sharding if you prefer that approach.

Reply:
fixed. pulling the endpoint information from request